Formula/bareos-fd.rb

```diff
@@ -0,0 +1,70 @@
+class BareosFd < Formula
+  desc "Open-Source Data Protection Solution"
+  homepage "https://bareos.org"
+  url "https://github.com/bareos/bareos/archive/Release/16.2.7.tar.gz"
+  sha256 "0b8dbe2e17b3eda470c30d4fe6af92e6f7b668fd1cfc1b045becdc8c8b6a767f"
+
+  depends_on "readline"
+  depends_on "openssl"
+  depends_on :python => :build
+
+  def install
+    system "./configure",
+    "--prefix=#{prefix}",
+    "--sbindir=#{bin}",
+    "--with-working-dir=#{var}/lib/bareos",
+    "--with-archivedir=#{var}/bareos",
+    "--with-confdir=#{etc}/bareos",
+    "--with-configtemplatedir=#{lib}/bareos/defaultconfigs",
+    "--with-scriptdir=#{lib}/bareos/scripts",
+    "--with-plugindir=#{lib}/bareos/plugins",
+    "--with-fd-password=XXX_REPLACE_WITH_CLIENT_PASSWORD_XXX",
+    "--with-mon-fd-password=XXX_REPLACE_WITH_CLIENT_MONITOR_PASSWORD_XXX",
+    "--with-basename=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
+    "--with-hostname=XXX_REPLACE_WITH_LOCAL_HOSTNAME_XXX",
+    "--with-python",
+    "--enable-client-only"
+
+    system "make"
+    rm "platforms/osx/Makefile"
```

Comment:
What's the deal with this? It looks like an upstream bug

Reply:
More or less. The file platforms/osx/Makefile is intended to create a PKG file, which obviously doesn't make sense to as a Homebrew build.

Reply:
I would propose to leave this like it is. However, a comment before the remove might be useful.